### PR TITLE
Add ';' shortcut for "Add comment"

### PR DESCRIPTION
**Pull request by Properrr**
Implemented of:
#11

### CODE REVIEW

**Comment by xarkes** [comment]
I just add a quick look, does it work with the DisassemblerGraphView?

**Comment by xarkes** [comment]
ping ?

**Comment by Properrr** [comment]
@xarkes yep, i reopen this PR with new algos

**Comment by Maijin** [comment]
no need to reopen a new PR, just push -f here

**Comment by xarkes** [comment]
Yes just push -f

**Comment by xarkes** [comment]
Please rebase

File: src/widgets/DisassemblyWidget.cpp [hunk 1]
```diff
@@ -50,18 +50,9 @@ DisassemblyWidget::DisassemblyWidget(QWidget *parent)
     connect(mDisasTextEdit, SIGNAL(customContextMenuRequested(const QPoint &)),
             this, SLOT(showDisasContextMenu(const QPoint &)));
 
-    // Scrollbar
-    connect(mDisasTextEdit->verticalScrollBar(), SIGNAL(valueChanged(int)), this, SLOT(disasmScrolled()));
```
Comment (xarkes) [inline src/widgets/DisassemblyWidget.cpp]: Why did you remove this?

Comment (xarkes) [reply src/widgets/DisassemblyWidget.cpp]: nvm

**Comment by xarkes** [comment]
Fixes #11
